test/runtime/configMatters/forall-unordered-opt/check-fence/check-fence-after-forall.skipif

```diff
@@ -0,0 +1 @@
+COMPOPTS <= --llvm
```

Comment:
We have these "grep the generated code" tests for other things, but they seem kinda fragile as we start seriously considering making llvm the default

Reply:
Yes, but once we are using LLVM, we'll use a FileCheck based check that looks at the LLVM IR. I'm not ready to do that for a case like this until we make LLVM the default, so here we are.